parsl/monitoring/monitoring.py

```diff
@@ -304,17 +303,17 @@ def close(self):
             self.logger.debug("Finished waiting for DBM termination")
 
     @staticmethod
-    def monitor_wrapper(f,
-                        try_id,
-                        task_id,
-                        monitoring_hub_url,
-                        run_id,
-                        logging_level,
-                        sleep_dur):
+    def monitor_wrapper(f: Any,
+                        try_id: int,
+                        task_id: int,
+                        monitoring_hub_url: str,
+                        run_id: str,
+                        logging_level: int,
+                        sleep_dur: int) -> Callable:
```

Comment:
This should be `float`? the type for `MonitorHub` is currently `float`

Reply:
yeah, probably. I've changed that